lib/knife-zero/devpatch/train_connector.rb

```diff
@@ -0,0 +1,25 @@
+require 'chef/knife/bootstrap/train_connector'
+require 'knife-zero/helper'
+
+class Chef
+  class Knife
+    class Bootstrap < Knife
+      class TrainConnector
+        class_eval do
+          def connect!
+            # Force connection to establish
+            connection.wait_until_ready
+            if connection.is_a? Train::Transports::SSH::Connection
+              connection.instance_variable_get(:@session).
```

Comment:
__[RuboCop]__ Layout/DotPosition: Place the . on the next line, together with the method name. ([link](https://sider.review/gh/repos/23179315/pulls/125#issue-4171564))

<sub>posted by Sider</sub>